appeals/web/src/server/appeals/appeal-details/appeal-timetables/__tests__/__snapshots__/appeal-timetables.test.js.snap

```diff
@@ -108,13 +108,7 @@ exports[`Appeal Timetables should render "Change Final Comment Review Date" with
                     <h2 class="govuk-error-summary__title"> There is a problem</h2>
                     <div class="govuk-error-summary__body">
                         <ul class="govuk-list govuk-error-summary__list">
-                            <li><a href="#due-date-day">Date day cannot be empty</a>
-                            </li>
-                            <li><a href="#due-date-month">Date month cannot be empty</a>
-                            </li>
-                            <li><a href="#due-date-year">Date year cannot be empty</a>
-                            </li>
-                            <li><a href="#">Date must be a valid date</a>
+                            <li><a href="#">Date must include a day, a month and a year</a>
```

Comment:
Isn't this a functionality nerf because these won't link to the inputs anymore?

Reply:
Yeah I guess that part of it won't work anymore 🤔 

It could probably be improved so the error links to the first field that's missing.

Reply:
I'm going to raise this with Naomi, we should probably create a ticket for this.